versions.nix

```diff
@@ -1,5 +1,9 @@
 {version}: let
   versionHashes = {
+    "9.5" = {
+      portable = "36abf5dbc510ef98e5139f776806b723d3bcc705";
+      openbsd = "f89bbe32af942a4dac8d2f9c13058ff63afa11d3";
```

Comment:
Hm, this latest commit seems to come after the version dump. Isn't `15da3dcc247333770eddcdc35238bbae3bc291c3` the hash we want? Looking at https://github.com/rpki-client/rpki-client-openbsd/commits/master/

Reply:
I took the one from the 9.5 [release tag](https://github.com/rpki-client/rpki-client-openbsd/tags). I assume they tagged that one since it's a fix on `15da3d`?

Reply:
Ah, I didn't see the tags, looks like this is the correct hash then 👍